packages/sdk/src/Actions/Action.ts

```diff
@@ -22,12 +22,12 @@ export type Action = EventAction; // | ContractAction | ERC721MintAction
 /**
  * A map of Action component interfaces to their constructors.
  *
- * @type {{ "0x7687b0ed": EventAction; }}
+ * @type {{ "0x6c3129aa": EventAction; }}
  */
 export const ActionByComponentInterface = {
   // ['0x6c3129aa']: ContractAction,
   // ['0x97e083eb']: ERC721MintAction,
-  ['0x7687b0ed']: EventAction,
+  ['0x6c3129aa']: EventAction,
```

Comment:
The contract didn't change so is the interface we published with wrong?

Reply:
yeah